src/WebJobs.Script/GlobalSuppressions.cs

```diff
@@ -82,4 +82,3 @@
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic", Scope = "member", Target = "Microsoft.Azure.WebJobs.Script.Description.ScriptFunctionInvokerBase.#InitializeEnvironmentVariables(System.Collections.Generic.Dictionary`2<System.String,System.String>,System.String,System.Object,System.Collections.ObjectModel.Collection`1<Microsoft.Azure.WebJobs.Script.Binding.FunctionBinding>,Microsoft.Azure.WebJobs.ExecutionContext)")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1822:MarkMembersAsStatic", Scope = "member", Target = "Microsoft.Azure.WebJobs.Script.Description.ScriptFunctionInvokerBase.#ProcessInputBindingsAsync(System.Object,System.String,Microsoft.Azure.WebJobs.Host.Bindings.Runtime.IBinderEx,System.Collections.ObjectModel.Collection`1<Microsoft.Azure.WebJobs.Script.Binding.FunctionBinding>,System.Collections.ObjectModel.Collection`1<Microsoft.Azure.WebJobs.Script.Binding.FunctionBinding>,System.Collections.Generic.Dictionary`2<System.String,System.String>,System.Collections.Generic.Dictionary`2<System.String,System.String>)")]
 [assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2000:Dispose objects before losing scope", Scope = "member", Target = "Microsoft.Azure.WebJobs.Script.Description.DotNetFunctionDescriptorProvider.#CreateFunctionInvoker(System.String,Microsoft.Azure.WebJobs.Script.Description.BindingMetadata,Microsoft.Azure.WebJobs.Script.Description.FunctionMetadata,System.Collections.ObjectModel.Collection`1<Microsoft.Azure.WebJobs.Script.Binding.FunctionBinding>,System.Collections.ObjectModel.Collection`1<Microsoft.Azure.WebJobs.Script.Binding.FunctionBinding>)")]
-
```

Comment:
Not sure why there was an additional line here.  I had to remove it to build successfully.